h/routes.py

```diff
@@ -127,7 +127,7 @@ def includeme(config):  # pylint: disable=too-many-statements
         traverse="/{id}",
     )
 
-    config.add_route("api.bulk", "/api/bulk", request_method="POST")
+    config.add_route("api.bulk.action", "/api/bulk", request_method="POST")
```

Comment:
Ok, so the `"api.bulk.action"` route is going to be for the existing API that creates things (creates users, groups, memberships, etc all in one API endpoint) and this creates an `"api.bulk.*"` namespace so that the new annotation getter API can be `"api.bulk.annotation"`.

I guess since the path for the new "action" endpoint is still just `"/api/bulk"` the route name could still be just `"api.bulk"` and then I think you could still use `"api.bulk.annotations"` for the new route name. But perhaps `"api.bulk.action"` is more explicit

Reply:
I'd like to move the end-point URL if possible, but it involves extra work, and definitely sounds like a job for another day. We could add both and then slide over?